compiler/ir/backend.common/src/KotlinIr.proto

```diff
@@ -74,6 +74,11 @@ message IrDeclarationOrigin {
   }
 }
 
+message Name {
```

Comment:
Why isn't it enough to encode names as strings with `Name.guessByFirstCharacter` as a way to distinguish special names from identifiers?

Reply:
Should be enough, I suppose. Thanks.

Reply:
Moved to use Name.guessByFirstCharacter().